cluster-scope/base/core/namespaces/connaisseur/kustomization.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: connaisseur 
+resources:
+- namespace.yaml
+components:
+- ../../../../components/project-admin-rolebindings/operate-first
```

will be also good to add these to list.
```suggestion
- ../../../../components/project-admin-rolebindings/operate-first
- ../../../../components/project-admin-rolebindings/octo-training-model
- ../../../../components/monitoring-rbac
- ../../../../components/resourcequotas/small
- ../../../../components/limitranges/default
```